hooks/useDelayedDisplay.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+
+const useDelayedDisplay = (delay: number) => {
```

Comment:
this seems to be doing very similar things to our existing `<ClientOnly>` component (since it will also only ever turn true on the client) - can we instead just use that wherever we're using this?